### ISSUE TITLE
Expose all of typebox

### ISSUE THREAD

**Issue by mcollina** [fastify/fastify-type-provider-typebox#53]
### Prerequisites

- [X] I have written a descriptive issue title
- [X] I have searched existing issues to ensure the feature has not already been requested

### 🚀 Feature Proposal

I think we should expose all of typebox to remove the need to install typebox separately (and risk potential versions conflicts):

```typescript
import Fastify from 'fastify'
import { TypeBoxTypeProvider, Type } from '@fastify/type-provider-typebox'

const fastify = Fastify().withTypeProvider<TypeBoxTypeProvider>()

fastify.get('/', {
  schema: {
    body: Type.Object({
      x: Type.String(),
      y: Type.Number(),
      z: Type.Boolean()
    })
  }
}, (req) => {
  // The `x`, `y`, `z` types are automatically inferred
  const { x, y, z } = req.body
})
```

**Comment by RafaelGSS** [comment]
This would mean whenever typebox gets a minor/major we need to update this package. Wouldn't it make things a bit complicated? 

Also, I've discussed it with @sinclairzx81 a long time ago. I just couldn't find the issue reference.

**Comment by mcollina** [comment]
What's the guarantee that a new version of typebox won't break this package?

**Comment by RafaelGSS** [comment]
We won't have a guarantee, but I doubt a new major/minor would break it without significant changes. I mean, I'm ok with exposing all the typebox, I just think it would be problematic at some point.

**Comment by mcollina** [comment]
My point is that it will be problematic anyway when it reaches that stage, e.g., there is some fundamental incompatibility.

**Comment by mcollina** [comment]
We might as well save a couple of lines to our users.

**Comment by mcollina** [comment]
@RafaelGSS I think #63 is an example of why we should re-expose the whole of TypeBox here and just wrap it. In this way we can avoid the peerDependency.

Would you be ok to make this change?

**Comment by mcollina** [comment]
Fixed in #64
